ext/net/ops_tls.rs

```diff
@@ -1144,10 +1187,15 @@ pub async fn op_tls_handshake(
   state: Rc<RefCell<OpState>>,
   rid: ResourceId,
   _: (),
-) -> Result<(), AnyError> {
+) -> Result<TlsHandshakeInfo, AnyError> {
   let resource = state
     .borrow()
     .resource_table
     .get::<TlsStreamResource>(rid)?;
-  resource.handshake().await
+
+  resource.handshake().await?;
```

Comment:
Could you change `resource.handshake()` to return the `TlsHandshakeInfo` directly?

Reply:
Sure, done. I've pushed an update.